elasticgraph-graphql/lib/elastic_graph/graphql/filtering/filter_interpreter.rb

```diff
@@ -339,7 +339,10 @@ def process_list_count_expression(bool_node, expression, field_path)
         end
 
         def build_bool_hash(&block)
-          bool_node = Hash.new { |h, k| h[k] = [] }.tap(&block)
+          # rubocop:disable Layout/LeadingCommentSpace
```

Comment:
RuboCop has `AllowRBSInlineAnnotation:` and `AllowSteepAnnotation:` configuration, but standardrb doesn't allow enabling it.

Reply:
I've just been using syntax like this:

```ruby
bool_node = Hash.new { |h, k| h[k] = [] } # : stringOrSymbolHash
```

Looks a little funny, perhaps, but works fine, with no need to enable/disable the cop.